src/TrixiAtmo.jl

```diff
@@ -36,7 +36,8 @@ export flux_chandrashekar, flux_LMARS
 export velocity, waterheight, pressure, energy_total, energy_kinetic, energy_internal,
        lake_at_rest_error, source_terms_lagrange_multiplier,
        clean_solution_lagrange_multiplier!
-export P4estCubedSphere2D, MetricTermsCrossProduct, MetricTermsInvariantCurl
+export P4estCubedSphere2D, P4estMeshQuadIcosahedron2D, MetricTermsCrossProduct,
```

Comment:
```suggestion
export P4estMeshCubedSphere2D, P4estMeshQuadIcosahedron2D, MetricTermsCrossProduct,
```

Reply:
We had this typo before this PR, but might as well fix it here since I just noticed it. Now `export P4estMeshCubedSphere2D` can be removed from the top of `meshes/meshes.jl` and all the `export` statements can be in the top-level `TrixiAtmo.jl` file.

Reply:
You're right. Done in 1bbb30784f7a772d140c45b7a38acb8d0154b2d0.